.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: scala
```

Comment:
Is this file still needed?  If so, please could you explain the reason for these alterations

Reply:
It's not needed if you don't care for Travis. We have it setup for many of our public repos for the early validation of PRs etc: https://travis-ci.org/github/hmrc/auth-client

Reply:
Thanks, Tim - we can keep it.